javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/reflectionmodel/ReflectionMethodResolutionLogic.java

```diff
@@ -117,9 +117,11 @@
         if (scopeType.getAncestors().isEmpty()) {
             Optional<ResolvedReferenceTypeDeclaration> optionalObjectClass = new ReferenceTypeImpl(new ReflectionClassDeclaration(Object.class, typeSolver), typeSolver).getTypeDeclaration();
             if (optionalObjectClass.isPresent()) {
-                SymbolReference<ResolvedMethodDeclaration> ref = MethodResolutionLogic.solveMethodInType(optionalObjectClass.get(), name, argumentsTypes);
-                if (ref.isSolved()) {
-                    MethodUsage usage = replaceParams(typeParameterValues, optionalObjectClass.get(), ref.getCorrespondingDeclaration());
+                Optional<ResolvedMethodDeclaration> ref = MethodResolutionLogic
+                        .solveMethodInType(optionalObjectClass.get(), name, argumentsTypes)
+                        .getCorrespondingDeclaration();
+                if (ref.isPresent()) {
```

Comment:
This reads as a change in behaviour -- going from "is solved" to "is present".

Looking at the code, I can see that these methods are equivalent.

Any ideas I come up with to change it seem overly verbose / introduce repeated checks (e.g., wrapping in `ifPresent` then doing the `isSolved` inside that (which then again calls `isPresent`) which would be), but if you can think of a quick/easy way around this please go for it.

Just a comment to say "could this be improved?", not a blocker :+1: